content/courses/Solid.md

```diff
@@ -0,0 +1,19 @@
+---
+title: SOLID Principles
+template: "course"
+draft: false
+slug: "/courses/Solid/"
+category: "Object-Oriented Design"
+tags:
+- "Usability"
+- "Maintainability"
+- "OOP"
+description: Familiarize yourself with the SOLID Design Principles and put them into practice. In the first lesson, walk through a conversational multiple-choice "quiz" (great for absolute beginners), and in the second, rewrite principle-violating code in line with the principles.
```

Comment:
I would recommend adding that this course assumes some basic familiarity with Java, though it doesn't have to be expert understanding.